src/main/java/org/sciborgs1155/lib/NoteVisualizer.java

```diff
@@ -0,0 +1,72 @@
+package org.sciborgs1155.lib;
+
+import static org.sciborgs1155.robot.pivot.PivotConstants.*;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.ScheduleCommand;
+import java.util.Set;
+import java.util.function.DoubleSupplier;
+import java.util.function.Supplier;
+import monologue.Annotations.Log;
+import org.sciborgs1155.robot.Constants.Field;
+import org.sciborgs1155.robot.pivot.PivotConstants;
+
+public class NoteVisualizer {
```

Comment:
`NoteVisualizer` should be static so that it's accessible from anywhere (eg. SimIntake)

Reply:
unironically i had a dream where i replied to this saying this was stupid and impossible and then you responded with something that suddenly made everything make sense but that's literally impossible so idk what you said in my dream but i imagine it must have been something wonderful